### DISCUSSION TITLE
Web-based editor for non cloud GitHub Enterprise

### DISCUSSION THREAD

**Discussion by jasonycw** [community/community#17352] [Enterprise]
Recently I discovered this great new feature on GitHub called "[web-based editor](https://docs.github.com/en/enterprise-cloud@latest/codespaces/the-githubdev-web-based-editor)"

- Press `.` while browsing any repository or pull request on GitHub

This is so helpful on browsing through repo between files without the need to open many browser tabs and "Go to file" every time or pull the whole repo to local just to have a look at the project structure easily.

Is there any plan to bring this to GitHub Enterprise? I saw that it is already available on [Enterprise Cloud](https://docs.github.com/en/enterprise-cloud@latest/get-started/using-github/keyboard-shortcuts#source-code-editing) but not in the current latest [Enterprise Server 3.5](https://docs.github.com/en/enterprise-server@3.5/admin/release-notes)

**Comment by sayyidusyofa** [comment]
We are considering to move to azure devops if this is not implemented
